apps/ledger-live-desktop/src/renderer/App.tsx

```diff
@@ -62,6 +62,14 @@ const InnerApp = ({ initialCountervalues }: { initialCountervalues: CounterValue
     return () => window.removeEventListener("keydown", reload);
   }, [reloadEnabled]);
 
+  useEffect(() => {
+    if (process.env.NODE_ENV === "development") {
+      const devTools = document.createElement("script");
+      devTools.src = "http://localhost:8097";
+      document.head.append(devTools);
+    }
+  }, []);
+
```

Comment:
I don't think that a `useEffect` is a best place to initialize React DevTools. We could do it in the render/init.tsx file and I think that you can use the `getEnv` method